cli/smartcontract/smart_contract.go

```diff
@@ -7,10 +7,12 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"path"
 	"path/filepath"
 	"strings"
 
+	"github.com/urfave/cli"
+	"gopkg.in/yaml.v2"
+
```

Comment:
:astonished:

Reply:
Are you aware of any linter\filewatcher that could help me with these empty lines?